jwst/stpipe/step.py

```diff
@@ -23,6 +23,8 @@
 except ImportError:
     DISCOURAGED_TYPES = None
 
+from crds.core import exceptions
```

Comment:
To avoid circular dependencies, use `from jwst.stpipe.crds_client import exceptions`